lib/verifier.js

```diff
@@ -101,7 +101,7 @@ Verifier.checkTxProposalBody = function(credentials, txp) {
     var proposalHeader = {
       outputs: outputs,
       message: txp.encryptedMessage || txp.message || null,
-      payProUrl: txp.payProUrl || undefined
+      payProUrl: txp.payProUrl
```

Comment:
when i experimented with replacing 'undefined' with 'null' here, sendTxProposal fails when the client passes in payProUrl = undefined. i think the root cause of the problem is that line 104 of verifier.js needs to match line 836 of api.js.